app/internal/window/window.go

```diff
@@ -64,6 +63,9 @@ type Driver interface {
 	// SetCursor updates the current cursor to name.
 	SetCursor(name pointer.CursorName)
 
+	// SetOption updates the current window Option.
+	SetOption(option Option)
```

Comment:
I prefer the "Options" name for the method; it matches Window.Options and "Set" implies replacing the options, not updating them.

// Options applies options to the window.
Options(options Option)